tools/msp_merge/mergeMSP.xml

```diff
@@ -0,0 +1,49 @@
+<tool id="mergemsp" name="Merge MSP Spectra" version="0.1.0" python_template_version="3.5">
+   <creator>
+        <person
+            givenName="Wudmir"
+            familyName="Rojas"
+            url="https://github.com/wverastegui"
+            identifier="0000-0001-7036-9987" />
+        <person
+            givenName="Helge"
+            familyName="Hecht"
+            url="https://github.com/hechth"
+            identifier="0000-0001-6744-996X" />
+        <organization
+            url="https://www.recetox.muni.cz/"
+            email="GalaxyToolsDevelopmentandDeployment@space.muni.cz"
+            name="RECETOX MUNI"/>
+    </creator>
+    <description>Merge MSP Spectra files</description>
+    <requirements>
+         <requirement type="package" version="0.14.0">matchms</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+       #set newinput = str("' '").join([str($f) for $f in $input])
+        python3  $__tool_directory__/mergeMSP.py
+        --filenames    '$newinput'
+        --outfilename  '$outfile'
+    ]]></command>
+    <inputs>
+        <param name="input" type="data" format="msp" multiple="true"/>
+    </inputs>
+    <outputs>
+        <data name="outfile" format="msp" />
+    </outputs>
+    <tests>
+         <test>
+            <param name="input">
+              <collection type="paired">
```

Comment:
Sure that this shouldn't be type `list`?

Reply:
@hechth, maybe `paired` is justified since the elements of this collection are static? Also, does it make any practical difference in this case?

Reply:
@hechth and @maximskorik, both cases show same behavior.

Reply:
Though at least conventionally it should be the `list` type. See [here](https://planemo.readthedocs.io/en/latest/writing_advanced.html#id2).